packages/ember-views/lib/mixins/dom_support.js

```diff
@@ -0,0 +1,285 @@
+/**
+@module ember
+@submodule ember-views
+*/
+
+import jQuery from "ember-views/system/jquery";
+
+import { Mixin } from "ember-metal/mixin";
+import { guidFor } from "ember-metal/utils";
+
+var DOMSupport = Mixin.create({
+  init() {
+    this._super(...arguments);
+
+    if (!this.elementId) {
+      this.elementId = guidFor(this);
+    }
+  },
+
+  /**
+    Appends the view's element to the specified parent element.
+
+    If the view does not have an HTML representation yet, `createElement()`
+    will be called automatically.
+
+    Note that this method just schedules the view to be appended; the DOM
+    element will not be appended to the given element until all bindings have
+    finished synchronizing.
+
+    This is not typically a function that you will need to call directly when
+    building your application. You might consider using `Ember.ContainerView`
+    instead. If you do need to use `appendTo`, be sure that the target element
+    you are providing is associated with an `Ember.Application` and does not
+    have an ancestor element that is associated with an Ember view.
+
+    @method appendTo
+    @param {String|DOMElement|jQuery} A selector, element, HTML string, or jQuery object
+    @return {Ember.View} receiver
+  */
+  appendTo(selector) {
+    var target = jQuery(selector);
```

Comment:
I'm curious for 2.x and browser support for IE9+ why not use `querySelector` or `querySelectorAll` instead of jQuery?